CountlyTests/CountlyBaseTestCase.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  CountlyBaseTestCase.swift
+//  CountlyTests
+//
+//  Created by Muhammad Junaid Akram on 27/12/2023.
+//  Copyright © 2023 Alin Radut. All rights reserved.
+//
+
+import XCTest
+@testable import Countly
+
+class CountlyBaseTestCase: XCTestCase {
+    var countly: Countly!
+    var deviceID: String = ""
+    let appKey: String = "58594c9a3f461ebc000761a68c2146659ef75ea0"
+    var host: String = "https://master.count.ly/"
+    
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+        cleanupState()
+    }
+    
+    func startCountly(deviceID: String? = nil) {
```

Comment:
those functions might be extracted to the utility ones, for example creating a config will be used probably in all test cases. And probably there is no need for one lines functions